lib/Mir.ml

```diff
@@ -93,7 +93,7 @@ type 's prog =
   ; prog_path: string }
 [@@deriving sexp]
 
-type stmt_loc = {sloc: string; stmt: stmt_loc statement}
+type stmt_loc = {sloc: string sexp_opaque; stmt: stmt_loc statement}
```

Comment:
Realistically, we'll also want to derive a `compare` operation on MIRs for the purpose of optimizations. In that case, you also want to add `[@compare.ignore]` to the location fields, I would guess.

Reply:
Woops, didn't see this in time. Will add to the next PR.